server/catgenome/src/main/java/com/epam/catgenome/util/feature/reader/TribbleIndexedFeatureReader.java

```diff
@@ -344,17 +347,20 @@ private void readHeader() throws IOException {
          *
          * @throws IOException
          */
-        public WFIterator() throws IOException {
+        WFIterator() throws IOException {
             final InputStream inputStream = ParsingUtils.openInputStream(path);
 
             final PositionalBufferedStream pbs;
+            final int bufferedStreamSize = 512000;
```

Comment:
Constants should be `public static final` and should comply to the `CONSTANT_NAMING_POLICY`. Please fix here and everywhere else.

Reply:
Changed.